packages/Tabs/src/Tabs.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import TabsContext from "./TabsContext";
+import Panel from "./components/Panel/Panel";
+import Panels from "./components/Panels/Panels";
+import Tab from "./components/Tab/Tab";
+import List from "./components/List/List";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  defaultIndex: PropTypes.number,
+  isDisabled: PropTypes.bool,
+};
+
+const defaultProps = {
+  defaultIndex: 0,
+  isDisabled: false,
+};
+
+const Tabs = props => {
+  const [activeIndex, setActiveIndex] = React.useState(props.defaultIndex);
+  const [currentFocusIndex, setFocusIndex] = React.useState(props.defaultIndex);
```

Comment:
Something is a little off with the `currentFocusIndex`... when your focus leaves the tabs, then comes back to the tabs, the focus should be reset to this value (currently it seems to reset to the first tab again, but then when you press an arrow key, it moves the focus relative the saved state of `currentFocusIndex`).